docs/openapi-docs/src/test/scala/sttp/tapir/docs/openapi/VerifyYamlTest.scala

```diff
@@ -29,6 +29,21 @@ class VerifyYamlTest extends AnyFunSuite with Matchers {
     .out(jsonBody[FruitAmount])
     .out(header[Int]("X-Role"))
 
+  test("should result in not parameter-fixed spec") {
```

Comment:
[minor] I'd add this as the last test :)

Also the description is a bit misleading, sth like: "should use usage-site, not declaration-site type parameter names" should be better